policyengine_us/variables/gov/states/az/tax/income/exemptions/az_parents_grandperents_exemption.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class az_parents_grandparents_exemption(Variable):
```

Comment:
the parents and grandparents are eligible for the exemption if they 
a) lived with the filer
b) the filer payed more than 1/2 of the total care and support costs. 
We have tot check these conditions for each parent / grandparent before they are eligible

Reply:
have no idea how to check whether the parents/grandparents are living with the filers, and also for the cost, what variables could be used here?

Reply:
we also need to create a variable for care_and_support_costs